### PR TITLE
Attempting upgrade of Grunt - not quite working

### PR DESCRIPTION
**Pull request by marktheunissen**
@elliotttf Could you look at this please? I have got it to 'work' in that I can run `grunt release` from the command line and it tells me that it all succeeds. However, some of the tasks maybe aren't working? Perhaps I did this wrong?

For reference:

http://gruntjs.com/upgrading-from-0.3-to-0.4

### CODE REVIEW

File: controller/web/Gruntfile.js [hunk 1]
```diff
@@ -5,11 +5,8 @@ module.exports = function(grunt) {
   grunt.initConfig({
     clean: ['dist/'],
 
-    lint: {
-      files: ['grunt.js', 'js/src/**/*.js']
-    },
-
     jshint: {
+      files: ['Gruntfile.js', 'js/src/**/*.js'],
```
Comment (elliotttf) [inline controller/web/Gruntfile.js:9]: is Gruntfile.js a magic thing now?

Comment (marktheunissen) [inline controller/web/Gruntfile.js:9]: not sure, but grunt.js was renamed Gruntfile.js, so I assumed we have to rename references to it.

Comment (elliotttf) [inline controller/web/Gruntfile.js:9]: lol, so it was, thanks.

**Comment by elliotttf** [comment]
Something's not quite right about the concat job. Digging in :)

**Comment by elliotttf** [comment]
Horray, the problem was that the job didn't work correctly when used as it was documented with the old version of grunt. Now with the new version of grunt it does work correctly as it's documented on the plugin's page. Moving on...

**Pull request merged**